pages/standards.tsx

```diff
@@ -111,6 +113,16 @@ const Standards = (data: StandardsProps) => {
                                         Imaging
                                     </Link>
                                 </li>
+                                <li>
+                                    <Link href="https://sage-bionetworks.github.io/schema_visualization/">
+                                        <a>
+                                            <FontAwesomeIcon
+                                                icon={faExternalLinkAlt}
+                                            />{' '}
+                                            HTAN schema visualization
```

Comment:
Let's swap the order of the text and the icon for consistency. For example see the `Manual` linkout.

![htan_manual_linkout](https://user-images.githubusercontent.com/3604198/206276986-4a4da2e7-cb4f-470d-9c43-2bbba49e1433.png)

```suggestion
                                            HTAN schema visualization{' '}
                                            <FontAwesomeIcon
                                                icon={faExternalLinkAlt}
                                            />
```

Reply:
Thanks @onursumer ! I just updated the code.